packages/tools/devtools/devtools-view/src/components/AudienceStateTable.tsx

```diff
@@ -115,7 +115,11 @@ export function AudienceStateTable(props: AudienceStateTableProps): React.ReactE
 							</TableCell>
 							<TableCell>{item.mode}</TableCell>
 							<TableCell>
-								<span>{item.scopes.join("\n")}</span>
+								<span>
+									{item.scopes.map((each_scope, index) => (
+										<li key={index}>{each_scope}</li>
+									))}
+								</span>
```

Comment:
Replace <span> to <ul>. Seems like it looks the same.

Reply:
I'd expect no visual change because browsers need to handle all sorts of funky HTML, yeah; it's just to make sure we're rendering correct HTML :). Maybe there warnings in the console.

Reply:
Seems like `AudienceStateView` has a warning: `Warning: validateDOMNesting(...): <button> cannot appear as a descendant of <button>`. But it's not due to this change, Taking a look to it...